src/app/sections/ProbeDetails.svelte

```diff
@@ -145,39 +154,32 @@ h2 {
 <div in:fly={paneTransition} class="drawer-section-container probe-details">
     <!-- probe-details-content -->
     <div class="probe-details-content">
+        <div class="probe-details-intro">
         {#if $store.probe.type}
-            <div class="drawer-section probe-labels">
-                <div>
-                    <span
-                        style="display: inline-block;"
-                        class="label label-text--01 label--{$store.probe.type}">{$store.probe.type}</span>
-                </div>
+            <dl class="drawer-section info-type">
+                <dt>{$store.probe.type}</dt>
                 {#if $store.probe.kind}
-                    <div class="probe-kind label-text--01">
-                        {$store.probe.kind}
-                    </div>
+                    <dd>{$store.probe.kind}</dd>
                 {/if}
+            </dl>
+        {/if}
+        {#if $store.probe.active !== undefined}
+            <div class="info-content">
```

Comment:
I'm wondering if we might be able to find more semantic names for `info-type` and `info-content` since semantically these bits of information are quite similar. Seems like they're just sort of left / right overview content, since it's less about whether it's `type` or `content` and more about handling the positioning. What do you think?